Brokerages/Zerodha/ZerodhaBrokerage.cs

```diff
@@ -0,0 +1,1096 @@
+﻿using Newtonsoft.Json;
+using QuantConnect.Data;
+using QuantConnect.Data.Market;
+using QuantConnect.Interfaces;
+using QuantConnect.Logging;
+using QuantConnect.Packets;
+using QuantConnect.Securities;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using NodaTime;
+using QuantConnect.Orders.Fees;
+using System.Threading;
+using QuantConnect.Orders;
+using QuantConnect.Brokerages.Zerodha.Messages;
+using Newtonsoft.Json.Linq;
+
+namespace QuantConnect.Brokerages.Zerodha
+{
+    /// <summary>
+    /// Zerodha Brokerage implementation
+    /// </summary>
+    public partial class ZerodhaBrokerage : Brokerage, IDataQueueHandler, IHistoryProvider
+    {
+        #region Declarations
+        /// <summary>
+        /// The list of queued ticks
+        /// </summary>
+        public List<Data.Market.Tick> Ticks = new List<Data.Market.Tick>();
```

Comment:
I believe this list is not required anymore, previously we used to add new ticks here but now we call this:
`_aggregator.Update(tick)`

All existing brokerages have been updated to use this method